.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
```

Comment:
where does this come from?
why changing it?

Reply:
I did not use this file.